src/core/storage/compression/Decompressor.h

```diff
@@ -50,7 +52,9 @@ class DecompressorImpl {
 	explicit DecompressorImpl(FILE *compressed) : m_compressed(compressed) {}
 
 	/// \brief The main function of decompression class that do all the magic with provided m_files.
-	void operator()() {
+	///
+	/// \param folder_name name of the folder to decompress (by default it is set to decompress all the files)
+	void operator()(const std::string &folder_name = "") {
```

Comment:
Please use `std::string_view` when the data is immutable.

Reply:
What is the difference, because I use this syntax in many other cases.